README.md

```diff
@@ -1,40 +1,106 @@
 
-# Quick Start
+<p align="center">
+  <img alt="GitHub" src="https://img.shields.io/github/license/701-T4/flatshare?style=flat-square">
+  <img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/701-T4/flatshare/PR%20Check?label=tests&logo=github&style=flat-square">
+  <img alt="GitHub repo size" src="https://img.shields.io/github/repo-size/701-T4/flatshare?style=flat-square">
+  <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/701-T4/flatshare?logo=github&style=flat-square">
+</p>
 
-0. Ensure you have NodeJS 16.x installed and it is your current node executable.
-1. Run `yarn` to fetch dependencies.
-2. Run `yarn run start:dev` to simultaneously run the frontend and backend.
+<p align="center">
+  <a href="#key-features">Features</a> •
+  <a href="#installation">Installation</a> •
+  <a href="#contributing">Contributing</a> •
+  <a href="https://github.com/701-T4/flatshare/wiki">Documentation</a> •
+  <a href="https://github.com/701-T4/flatshare/wiki/Trouble-Shooting">Troubleshooting</a> •
+  <a href="https://flat-split.web.app/">Demo</a>
+</p>
 
-# Running with Docker
+> **Disclaimer**
+>
+> This is a project for the University of Auckland SE701 course (Team 4). It will likely not be maintained after completion of this course.
 
-Running with Docker will ensure that users can quickly get environment up and running so they can quickly iterate on work. The application will work regardless of operating system and configuration.
+## Screenshots
 
-## Web app
+<!--https://felixhayashi.github.io/ReadmeGalleryCreatorForGitHub/-->
 
-Ensure you have set up backend env files first. Checkout backend README for more info.
+<img src="./resources/sc-1.png?raw=true" width="50%" alt="dashboard screenshot"></img><img src="./resources/sc-2.png?raw=true" width="50%" alt="home screenshot"></img>
+<img src="./resources/sc-3.png?raw=true" width="50%" alt="login screenshot"></img>
 
-1. Run `yarn run start:docker` to simultaneously run the frontend and backend in docker containers.
+## Key Features
 
-## Tests
+- Bill Splitting
+- Chores Management
+- Synchronised Notes
 
-1. Run `yarn run test:docker` to simultaneously run the frontend and backend tests in docker containers.
+## Installation
 
-For more detail, view `CONTRIBUTING.md`.
+To clone and run this application, you'll need [Git](https://git-scm.com), [Node.js 16.x](https://nodejs.org/en/download/), [Yarn](https://yarnpkg.com/getting-started/install) and [MongoDB](https://www.mongodb.com/try/download/community) installed on your computer. From your command line:
 
-# How to attribute
+```bash
+# Clone this repository
+$ git clone https://github.com/701-T4/flatshare
 
-## Attributing third party dependencies
+# Go into the repository
+$ cd flatshare
 
-All third party dependencies used in this project are attributed in the LICENSE-3RD-PARTY files. The files are generated automatically when there is an update to the dependecies used.
+# Install dependencies
```

Comment:
Do we have a preference or requirement to install frontend/backend dependencies only in their respective folders?

Reply:
that is the current implementation. If we change it later I will update the readme